src/components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Button from './Button';
+import ErrorCard from './ErrorCard';
+import Input from './Input';
+
+export default function Login() {
+  const [error, setError] = useState(null);
+  const [credentials, setCredentials] = useState({
+    email: '',
+    password: '',
+  });
+  const handleSubmit = async (e:any) => {
```

Comment:
I think theres something like a submit event for this. Can you check the attribute type?